lib/graphql/utilities/vanity/index.ts

```diff
@@ -13,12 +27,75 @@ interface VanityLinkPayload {
 }
 
 export async function generateVanityLink(payload: VanityLinkPayload) {
-  const stepFunction = createStepFunctionInstance();
+  // Depreciate step function 
+
+  // const stepFunction = createStepFunctionInstance();
+
+  // return stepFunction.send(new StartExecutionCommand({
+  //   stateMachineArn: process.env.VANITY_ARN!,
+  //   name: uuid(),
+  //   input: JSON.stringify(payload),
+  // }));
```

Comment:
If you're sure that your function is working, feel free to completely remove it instead of just comment it out.